src/main/java/uk/nhs/hee/trainee/details/config/RabbitConfig.java

```diff
@@ -49,6 +54,21 @@ public RabbitTemplate rabbitTemplate(final ConnectionFactory connectionFactory)
     final RabbitTemplate rabbitTemplate = new RabbitTemplate(connectionFactory);
     rabbitTemplate.setMessageConverter(jsonMessageConverter());
     rabbitTemplate.containerAckMode(AcknowledgeMode.AUTO);
+
+    ExponentialBackOffPolicy backOffPolicy = new ExponentialBackOffPolicy();
+    backOffPolicy.setInitialInterval(500);
+    backOffPolicy.setMultiplier(3.0);
+    backOffPolicy.setMaxInterval(3000_000);
+    RetryTemplate retryTemplate = new RetryTemplate();
+    retryTemplate.setBackOffPolicy(backOffPolicy);
+    SimpleRetryPolicy simpleRetryPolicy = new SimpleRetryPolicy();
+    simpleRetryPolicy.setMaxAttempts(15);
```

Comment:
This configuration takes effect if Rabbit is not available. One thing we might want to consider is that it is currently a blocking retry, so the front-end is left hanging, and will eventually time-out, if Rabbit is down. This is not ideal.